packages/overlay/src/integrations/sentry/components/traces/spans/SpanTree.tsx

```diff
@@ -36,6 +38,7 @@ export default function SpanTree({
             totalDuration={totalDuration}
             spanNodeWidth={spanNodeWidth}
             setSpanNodeWidth={setSpanNodeWidth}
+            query={query}
```

Comment:
I'm wondering if this should be passed in the context at this point. Have you considered that?

Reply:
hmm, yes we can use context here as well.
should i create one or we can fix it as a whole in sentry integration for other states as well in next PR?

Reply:
>should i create one or we can fix it as a whole in sentry integration for other states as well in a separate PR?

Separate PR as I understand there are more things we can move to context from what you say.